docs/installation.rst

```diff
@@ -68,21 +81,45 @@ Testing your installation
 
 Test your YANK installation to make sure everything is behaving properly on your machine:
 
-.. code-block:: none
+.. code-block:: bash
 
    $ yank selftest
 
 This will not only check that installation paths are correct, but also run a battery of tests that ensure any automatically detected GPU hardware is behaving as expected.
 
+|
+
+Optional Tools
+--------------
+
+The `OpenEye toolkit and Python wrappers <http://www.eyesopen.com/toolkits>`_ can be installed to enable free energy calculations to be set up directly from `multiple supported small molecule formats <https://docs.eyesopen.com/toolkits/python/oechemtk/molreadwrite.html#file-formats>`_, including
+
+* Tripos mol2
+* SDF
+* SMILES
+* IUPAC names
+* PDB
```

Comment:
`PDB` and `mol2` support does not depend on OpenEye if one doesn't want to use the OE charge scheme.